### DISCUSSION TITLE
Azure / Office / Teams theme?

### DISCUSSION THREAD

**Discussion by lonix1** [BlazorFluentUI/BlazorFluentUI#415] [Q&A] [answered]
I found this library, but am unsure what is "fluent" / "fabric".

I was searching for a blazor design library that has an azure / office365 / teams feel. Is this library similar to that?

**Comment by vnbaaij** [comment, marked as answer]
Fluent is the Microsoft design language for all its apps. Used to be called fabric before that. It originated from Office design.

You can certainly use this to get that Azure/Office feel in your Blazor app, but the library will not be getting any significant updates anymore (what is there is pretty good so doesn't have to be a problem) As an alternative I would recommend [Microsoft Fluent UI web components for Blazor library] (https://github.com/microsoft/fast-blazor) (I'm a maintainer there nowadays)

**Reply by lonix1** [reply]
Thank you that is very helpful!

From a quick reading of the "FAST" docs, it seems that Microsoft uses it internally for azure/office/teams products - exactly what I wanted.

**Reply by vnbaaij** [reply]
Great! See you in the other repo then (if/when questions/issues arise)

**Reply by lonix1** [reply]
@vnbaaij  One last thing, if I may - I've gone through the fast/fluent docs and all I saw was components.

Am I right in assuming that a layout system (containers, rows, columns, etc.) is not included, so I'm responsible for it?

So for example, I could use Bootstrap's layout system (e.g. via `BlazorStrap`), but use Fluent's components (via `Fast-Blazor`).

**Reply by vnbaaij** [reply]
Yes, it is only components. Layout is done with the standard CSS techniques like grids, etc.
